app.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const controllers = require('./controllers/app.controllers');
+
+const app = express();
+
+// some console feedback for when using the app from a terminal
+app.use((req, res, next) => {
```

Comment:
although this is cool, we do need to be careful how much superfluous stuff we have in our repo as is should be production ready and also to spec.

Reply:
purely for my benefit, i will remove it